### PR TITLE
docs(project-management): the creation bar names what never files

### PR DESCRIPTION
**Pull request by bmethod**
The 2026-08-27 backlog audit canceled 56 of 134 open issues. Every one had cleared the creation bar at filing time because the bar named only one exclusion ("a severe-sounding edge case that no real input reaches").

The bar is the one choke point: orch's filing bar defers to it, `tpm-audit` § 10.1 applies it, and direct creates are routed to the TPM pipeline. So the change is one sentence there, plus the audit's cancellation sweep re-reading active issues against it.

| Where | Change |
|---|---|
| SKILL.md § Disposition, creation bar | names what fails: hypothetical + low severity, unreached edge cases, coverage asks with no regression, refactors with no behavior change, same-machine race / crash-between-writes guards, review-bred mechanisms; critical harm and financial loss file at any likelihood |
| SKILL.md § Disposition, burn-down | the sweep also proposes active issues that fail the bar today |
| tpm-audit.md § 6.2 | "Below the bar": such issues go to `obsolete[]` with `{below_bar, test, who_hits_it}` so the report offers them as removal candidates with the user story |
| audit-output.md | the third `obsolete[]` evidence shape |

Source and committed render edited together. `disposition-contract.test.sh` passes.

https://claude.ai/code/session_01YMdDBjvjpv1qCSuFr69eS7

### CODE REVIEW

File: skills/project-management/workflows/tpm-audit.md [hunk 1]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
 
 With `DECISION_REF` present, also detect issues the decision made unnecessary by changing the approach (not the scope): read the decision (`.agents/skills/decider/scripts/decisions get [DECISION_REF]`), extract the patterns it explicitly replaced, search the comparison set for issues implementing them, exclude anything already in `supersedes[]`, and record confidence 100 with evidence `{decision_eliminated: true, decision_ref, eliminated_pattern}`.
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar as it stands today. One that fails it — a hypothetical of low severity, an edge case no real input reaches, a coverage ask with no regression, a refactor with no behavior change, a guard bred by a review round — goes to `obsolete[]` with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it. Critical-harm and financial-loss edge cases never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 0dd6f6ff

File: .agents/skills/project-management/SKILL.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ Wrappers run in the primary session: they own the user dialog and every tracker
...
-- **Burn down more than you create.** Any audit that proposes creations also sweeps its comparison set for issues the codebase has already satisfied, duplicated, or superseded, and proposes those for cancellation in the same pass. Report `created N / closed M`.
+- **Creation bar.** File an issue only when all three hold: it changes what a user or operator experiences, or blocks work that does; no open issue, active branch, or one-line fix already covers it; and someone could pick it up and finish it without a new investigation. A reproducible anomaly with evidence in hand passes all three as an investigation issue. Everything else is declined with one line in the report — no issue, no placeholder, no tracking artifact. A severe-sounding edge case that no real input reaches fails the first test, and so does a hypothetical of low severity, a coverage ask for a path that has not regressed, a refactor with no behavior change, a guard for a race between two invocations on one machine or a crash between two writes, and a mechanism that itself came out of a review round; the exception is an edge case whose failure is critical harm or financial loss, which files at any likelihood.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply .agents/skills/project-management/SKILL.md]: Fixed in 0dd6f6ff

File: skills/project-management/workflows/tpm-audit.md [hunk 3]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar as it stands today. One that fails it — a hypothetical of low severity, an edge case no real input reaches, a coverage ask with no regression, a refactor that neither changes behavior nor unblocks user-visible work, a guard bred by a review round — is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it. In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in f8617115

File: .agents/skills/project-management/workflows/tpm-audit.md [hunk 4]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar as it stands today. One that fails it — a hypothetical of low severity, an edge case no real input reaches, a coverage ask with no regression, a refactor that neither changes behavior nor unblocks user-visible work, a guard bred by a review round — is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it. In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply .agents/skills/project-management/workflows/tpm-audit.md]: Fixed in 6d85b7bd

File: skills/project-management/workflows/tpm-audit.md [hunk 5]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar's first and third tests as they stand today (the second, coverage by other work, is § 6.1's duplicate sweep and never counts an issue against itself or its own branch). One that fails — a hypothetical of low severity, an edge case no real input reaches, a coverage ask with no regression, a refactor that neither changes behavior nor unblocks user-visible work, a guard bred by a review round — is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it. In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 84f41d91

File: skills/project-management/SKILL.md [hunk 6]
```diff
@@ -23,8 +23,8 @@ Wrappers run in the primary session: they own the user dialog and every tracker
...
+- **Burn down more than you create.** Any audit that proposes creations also sweeps its comparison set for issues the codebase has already satisfied, duplicated, or superseded, and proposes those for cancellation in the same pass, along with every active issue that fails the creation bar as it stands today. Report `created N / closed M`.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/SKILL.md:27]: Fixed in 84f41d91

File: skills/project-management/workflows/tpm-audit.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 689c7f43

[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 689c7f43

File: .agents/skills/project-management/workflows/tpm-audit.md [hunk 7]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar's first and third tests as they stand today (the second, coverage by other work, is § 6.1's duplicate sweep and never counts an issue against itself or its own branch). One that fails (the bar's own list, not restated here) is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it. In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply .agents/skills/project-management/workflows/tpm-audit.md]: Fixed in 349f7dc4

File: skills/project-management/workflows/tpm-audit.md [hunk 8]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set is re-read against the creation bar's first and third tests as they stand today (the second, coverage by other work, is not reapplied: an issue always covers itself). One that fails (the bar's own list, not restated here) is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it, written after reading the issue's body (on GitHub, `gh issue view <n>`; the § 1.5 list carries no body). In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 349f7dc4

[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in 349f7dc4

File: skills/project-management/workflows/tpm-audit.md [hunk 9]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set as § 1.5 fetched it is re-read against the creation bar's first and third tests as they stand today (the second, coverage by other work, is not reapplied: an issue always covers itself). One that fails (the bar's own list, not restated here) is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it, written after reading the issue's body (on GitHub, `gh issue view <n>`; the § 1.5 list carries no body). In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here. The code-verification rule above is for implementation-obsolete entries; a `below_bar` entry is verified by reading the body and, where it names a path, producer, or regression, checking that claim in the repository before the entry is written.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply skills/project-management/workflows/tpm-audit.md]: Fixed in b2856a34

File: .agents/skills/project-management/workflows/tpm-audit.md [hunk 10]
```diff
@@ -206,6 +206,8 @@ Never add to `obsolete[]` without code verification.
...
 
+**Below the bar.** Every active issue in the comparison set as § 1.5 fetched it is re-read against the creation bar's first and third tests as they stand today (the second, coverage by other work, is not reapplied: an issue always covers itself). One that fails (the bar's own list, not restated here) is a cancellation with confidence 100 and evidence `{below_bar: true, test, who_hits_it}`: `test` names the failed test, `who_hits_it` is the one-line user story and how often a user meets it, written after reading the issue's body (on GitHub, `gh issue view <n>`; the § 1.5 list carries no body). In project mode it is an `obsolete[]` entry; in issue mode it is the issue's own `issues[]` entry with `action: "cancel"` and that evidence in its `obsolete` field. The bar's two exceptions (a shipped-path security or data-loss defect; a critical-harm or financial-loss edge case) never go here on likelihood; the third test still applies to them. The code-verification rule above is for implementation-obsolete entries; a `below_bar` entry is verified by reading the body and, where it names a path, producer, or regression, checking that claim in the repository before the entry is written.
```
[the remark this answers is not in the document]
Comment (bmethod) [reply .agents/skills/project-management/workflows/tpm-audit.md:209]: Declined: scope freeze on this PR (round 8, wording class). A one-line fix is the third test answered in the affirmative — someone can finish it — not grounds to cancel; the sweep cancels on likelihood and finishability, and the one-line-fix branch of the second test is a filing-time question, not an audit-time one.
